react-app/src/features/forms/post-submission/amend/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Navigate, useParams } from "react-router-dom";
+import { AmendmentForm as CapitatedForm } from "../../waiver/capitated";
+import { AmendmentForm as ContractingForm } from "../../waiver/contracting";
+import { useGetItem } from "@/api";
+
+export const Amendment = () => {
+  const { id } = useParams();
+  const item = useGetItem(id);
+  const isCapitated = item.data._source.changelog.find(
+    (event) =>
+      event._source.event === "capitated-initial" || event._source.event === "capitated-renewal",
+  );
+  const isContracting = item.data._source.changelog.find(
+    (event) =>
+      event._source.event === "contracting-initial" ||
+      event._source.event === "contracting-renewal",
+  );
+
+  if (isCapitated) {
+    return <CapitatedForm />;
+  } else if (isContracting) {
+    return <ContractingForm />;
+  } else {
+    // there is probably an error or someone is doing something they shouldn't
+    return <Navigate to="/dashboard" />;
+  }
```

Comment:
We can simplify this by removing the `else` and just rely on the return to break us out of the conditionals

Reply:
Sorry if I didn't make it clear enough, but we can yeet any `else if` by simply having:
```ts
if (something) {
  return <Something />
}
```